### PR TITLE
Fix extraneous constraints on tables

### PR DESCRIPTION
**Pull request by spolu**
Fixes: https://github.com/dust-tt/tasks/issues/165

Deploy and delete extraneous contraints in production on tables `messages`, `agent_dust_app_run_configurations`, `conversations`

### CODE REVIEW

File: front/lib/models/assistant/conversation.ts [hunk 1]
```diff
@@ -464,11 +462,11 @@ Message.init(
     indexes: [
       {
         unique: true,
-        fields: ["conversationId", "rank", "version"],
+        fields: ["sId"],
```
Comment (lasryaric) [inline front/lib/models/assistant/conversation.ts:465]: how is changing the order fixing it? I don't get it.

**Review by fontanierh — COMMENTED** [review]
I don't understand the diff

File: front/lib/models/assistant/conversation.ts (hunk 1 above)
Comment (spolu) [reply front/lib/models/assistant/conversation.ts:465]: see: https://github.com/dust-tt/tasks/issues/165#issuecomment-1786769114

Comment (lasryaric) [reply front/lib/models/assistant/conversation.ts:465]: ok got it. Given the unexpected behavior of sequelize, I thought the order mattered.
Thanks.

**Review by lasryaric — APPROVED** [review]

**Pull request merged**